### DISCUSSION TITLE
Reset the calendar state on close

### DISCUSSION THREAD

**Discussion by indranil** [wojtekmaj/react-daterange-picker#125] [Q&A]
Hello!

I am trying to restore the default dates if you close the calendar without selecting a complete range (for eg: you select start date, or you go 5 years down the line, then close the calendar), but I can't seem to find a way to reset the functionality easily.

Is this something that can be done with this package?

**Comment by wojtekmaj** [comment]
There's `onCalendarClose` callback that might help you out here. You would need to change value passed to React-DateRange-Picker on this callback.
